app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -1 +1,31 @@
 /* Placeholder for backend dummy app */
+SpreePayuIntegration = {
+  updateSaveAndContinueVisibility: function() {
+    if (this.isButtonHidden()) {
+      $(this).trigger('hideSaveAndContinue')
+    } else {
+      $(this).trigger('showSaveAndContinue')
+    }
+  },
+  isButtonHidden: function () {
+    paymentMethod = this.checkedPaymentMethod();
+    return (SpreePayuIntegration.paymentMethodID && paymentMethod.val() == SpreePayuIntegration.paymentMethodID);
+  },
+  checkedPaymentMethod: function() {
+    return $('div[data-hook="checkout_payment_step"] input[type="radio"][name="order[payments_attributes][][payment_method_id]"]:checked');
+  },
+  hideSaveAndContinue: function() {
+    $("#checkout_form_payment [data-hook=buttons]").hide();
+  },
+  showSaveAndContinue: function() {
+    $("#checkout_form_payment [data-hook=buttons]").show();
+  }
+}
+
+$(document).ready(function() {
+  SpreePayuIntegration.updateSaveAndContinueVisibility();
+  paymentMethods = $('div[data-hook="checkout_payment_step"] input[type="radio"]').click(function (e) {
```

Comment:
If your template is very custom and removes hooks you need to either implement this JS on your own (you do not need to include it at the first place) or add missing hooks back, simple as that.

Adding a small info to readme about this is a very good idea, will update.